getting_started.md

```diff
@@ -19,7 +21,21 @@ $ ./local_setup.sh
 $ npm run dev
 ```
 
-Then you can open http://localhost:8080/ (or http://local.findthemasks.com:8080) in your browser.  If everything is working you should be able to see the dynamically populated "List of Donation Sites"
+#### Running in docker
+
+Requirements:
+* docker
```

Comment:
@maplebird @awong-dev @drwestco Do we really need docker? I'm concerned that it's just adding complexity to something that doesn't need it.

For context I've just spent over 45 minutes trying to get this working since I didn't have docker on my laptop. I figured it'd be pretty straightforward but even after finally getting everything installed (brew install docker doesn't install docker-compose, needing to change security preferences, etc.) and getting `docker-compose up` to run, I get `localhost refused to connect.` when I try and open `localhost:3000`. I'm now in the process of uninstalling all that and trying to download Docker Desktop for Mac to see if that will work.

If someone has a strong reason why we need docker and can write detailed instructions on how to install it for different os' I'm open to it but I also don't want people to spend hours getting started.

Reply:
I agree, I think given how few requirements this app has (node 12, basically), there's no need for a docker development environment and since our current plan is to deploy to Heroku, it doesn't need to be dockerized for deployment either.

That said, what @maplebird has here is optional -- you don't have to use Docker to run it if you don't want @nburt

Reply:
OK, how about we just update the getting started to not mention docker but leave the config in for those who want to use it?

Reply:
What does docker buy us here? The changes to the readme mention auto-refresh when changing code, but running under nodemon does the same thing.

Reply:
I personally don't think we need docker (it's too much overkill), but I've seen several people ask for it and it only took about 10 minutes to do.  

Instead of keeping it in the repo, a compromise could be keeping Dockerfile and docker-compose in a wiki page so if someone does want to run this in docker, they can.

That said, I feel 80% of web devs probably have node installed.

@nburt oh yeah the docker you install with homebrew is complete and utter crap IMO, Docker Desktop is miles better.